apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -243,6 +243,19 @@ export default connect(
     }
 
     displayOutputToConsole() {
+      const inspectorTheme = {
+        ...chromeLight,
+        ...{
```

Comment:
nit: you can remove this nested object since we're using the spread operator as well:

```javascript
{
  ...chromeLight,
  OBJECT_VALUE_NULL_COLOR: 'rgb(88, 92, 246)',
}
```